app/controllers/dispatches_controller.rb

```diff
@@ -17,13 +17,13 @@ def create
   def update
     @operation_period = OperationPeriod.find(params[:operation_period_id])
     @dispatch = Dispatch.find(params[:id])
-    params[:dispatch][:communications].each do |communication|
-      if params[:dispatch][:communication_description][communication[0]].present?
-        @dispatch.asset_communications.create(communication_id: communication[1], dispatch_id: @dispatch.id, description: params[:dispatch][:communication_description][communication[0]])
-      end
-    end
+    @dispatch.update(dispatch_params)
+    # params[:dispatch][:communications].each do |communication|
```

Comment:
@andyhull I had to disable the `communications` on each asset's `update`. They were already gone from create.